src/platform-implementation-js/driver-common/sidebar/constants.ts

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames';
+import { legacyIdMap } from '../../lib/idMap';
+
+const cssGlobal = 'inboxsdk__app_sidebar_waiting_platform';
+
+const enum SidebarClassName {
+  /** v1.1.0 CSS module selector, */
+  v1_1_0 = 'inboxsdk__a0oshmeH6N_2vLtc3DNT',
+  /** v1.1.1 CSS module selector, */
+  v1_1_1 = 'inboxsdk__BV_FA4qVT4qYxs3LbEU4',
+}
```

Comment:
Weird to use an enum for this since "SidebarClassName" isn't really a type, and const enums have weird compatibility catches with Babel. Could this just be an object instead?

Reply:
> const enums have weird compatibility catches with Babel

Not true since the end of 2021.

https://github.com/babel/babel/pull/13324

An enum a shorthand to group the two values together and it compiles to an object at build time.

Reply:
Sure, but both the `const` and `enum` parts do nothing here.

Reply:
With [01238b2](https://github.com/InboxSDK/InboxSDK/pull/895/commits/01238b2e453b7e76fd8a2e8fee18cf892691b2fe), which I had in #885, the `const enum` is a zero cost abstraction and inlined at runtime.

Reply:
I think it's still weird because a "SidebarClassName" type makes no sense and doesn't get us anything more than an object would here, but this is non-blocking.